AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_relocator_tests.py

```diff
@@ -3785,6 +3788,7 @@ class TestsAssetProcessorMove_WindowsAndMac:
     # run all the tests for a test case:
     # -k C19462747
 
+    @pytest.mark.skipif(sys.platform.startswith('linux'), reason="https://github.com/o3de/o3de/issues/14514")
```

Comment:
Is there a reason you're using sys.platform.startswith to skip instead of the other method:
@pytest.mark.skipif(ly_test_tools.LINUX,

I'm guessing it's mostly information discovery - it took me a lot of digging to find the ly_test_tools.LINUX

Reply:
No particular reason. I wasn't aware of this method. It looks like it does the same thing behind the scenes.